README.md

```diff
@@ -216,7 +216,7 @@ which can be used for actual migration using the `--identifiers-from-file` optio
 
 Sample input:
 ```shell
-java -jar iceberg-catalog-migrator-cli-0.2.0.jar migrate \
+java -jar iceberg-catalog-migrator-cli-0.2.0.jar register \
```

Comment:
The below text descriptions has to be changed from `migaration` to `registration`, `migrate` to `register`, `migrated` to `registered`
These examples were captured before blocking migration on Hadoop. 

or easier way is to use HIVE as source catalog for these scenario examples. 

I can do it in follow up If you don't have time for this.

Reply:
Thanks, can you please review again now.